gnd/src/main/java/com/google/android/gnd/persistence/local/room/RoomLocalDataStore.java

```diff
@@ -268,7 +268,7 @@ public Completable applyAndEnqueue(FeatureMutation mutation) {
   @Override
   public Flowable<ImmutableSet<Tile>> getTilesOnceAndStream() {
     return tileDao
-        .findAll()
+        .getTilesOnceAndStream()
```

Comment:
let's stick with the "findAll" prefix for consistency with other DAOs, so "`findAllOnceAndStream`?